internal/run/wrapper.go

```diff
@@ -21,6 +21,9 @@ type API struct {
 	Region string
 }
 
+// regions are the available regions.
+var regions = []string{}
```

Comment:
can you add a `// TODO caching regions might be unnecessary if we are querying them once during the lifespan of the process`